wildfly@.service

```diff
@@ -21,6 +21,7 @@ ExecStartPre=/usr/bin/docker pull ${DOCKER_REPO}:${VERSION}
 ExecStart=/bin/bash -c 'docker run -P --name ${CONTAINER} --restart=always \
   --env ADMIN_USERNAME=$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/wildfly/admin-username?raw) \
   --env ADMIN_PASSWORD=$(curl -s http://${COREOS_PRIVATE_IPV4}:8500/v1/kv/wildfly/admin-password?raw) \
+  --env JAVA_OPTS="-Xms1024m -Xmx4096m -XX:MaxPermSize=4096m -Djava.net.preferIPv4Stack=true -Djboss.modules.system.pkgs=org.jboss.byteman -Djava.awt.headless=true" \
```

Comment:
Putting on a very dusty JVM tuning hat here:

In theory we could start with a smaller size that 1GB and let it grow. It might be a candidate for getting the size from consul so that in dev we can set them smaller and in prod bump them both up. I'm also highly suspicious of a maximum perm gen space the size of the entire heap. That would leave no room for application memory. Perm gen is where class definitions go (and some other things like string literals, though that might have changed). Anyway, a big perm gen helps when loading lots of applications like we do, or reloading when the unloading process is imperfect (as it usually seems to be). But I've always seen the perm gen be at most 1/4 of the JVM memory space. That could be some pretty old thinking, but I can't see how having no space for heap or stack being a good thing.